GettyImages.Api/WebHelper.cs

```diff
@@ -34,13 +34,7 @@ internal WebHelper(Credentials credentials, string baseAddress, DelegatingHandle
                         BuildQuery(queryParameters)
                 };
 
-                var httpResponse = await client.GetAsync(builder.Uri);
-
-                if ((int)httpResponse.StatusCode >= 500)
-                {
-                    Task.Delay(1000).Wait();
```

Comment:
Using the retry policy should eliminate the need for these manual delays as well.